### ISSUE TITLE
PostProcessor: cut logfiles

### ISSUE THREAD

**Issue by msb002** [aspitarl/MHDlab#41]
At this rate we're going to end up with 10 different cut functions for logfiles. I'm thinking we either generalize the log files (for logfiles which have channels equal length to the time channel, just cut the group by the channel titled "Time") or have it match the format of the tdms file so you dont have to find the cut function.

**Comment by aspitarl** [comment]
Yes, I was thinking that the post processor would automatically choose the function based on the filename using a regular expression.
